lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -123,7 +125,7 @@ var queryTemplate = Node.template([
     '_data AS ( ',
     '  SELECT id AS __obs_id__, {{=it.obsColumns}}',
     '  FROM cdb_dataservices_client._OBS_GetData_exception_safe(',
-    '    (SELECT ARRAY_AGG((the_geom, cartodb_id)::geomval) FROM _source),',
+    '    (SELECT geomvals FROM _summary),',
```

Comment:
BTW in the CTE you are generating a table with the extent duplicated for every row, so the WHERE clause is not duplicated but you'll have N copies of the extent polygon in memory, one per row.